rclcpp/test/rclcpp/test_qos_event.cpp

```diff
@@ -473,6 +521,10 @@ TEST_F(TestQosEvent, test_pub_matched_event_by_set_event_callback)
 
 TEST_F(TestQosEvent, test_sub_matched_event_by_set_event_callback)
 {
+  std::string rmw_implementation_str = std::string(rmw_get_implementation_identifier());
```

Comment:
matched events should definitely work else there is a bug in rmw_zenoh.

Reply:
This one is working